how-we-work/epic-leads.md

```diff
@@ -0,0 +1,80 @@
+# How to be an Epic Lead
+
+## Background
+
+Every quarter, each engineer will typically have at least one Objective that is their responsibility to lead, work on and land during the quarter. Each engineer will work with their manager to figure out what these Objectives are prior to the start of a quarter. Ignoring Technical Objectives for the time being, the Product Objectives' priorities are set be the Product team in the previous quarter based on Product Discovery and User Research and dictate the next set of features to ship. The priority designates the _order_ in which the Objective will be worked.
+
+In order to figure out when you will be called to "batter up" (i.e. become an Epic Lead to start leading the charge on your Objective), you can take a look at the Epic column in our [Zenhub board](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/board?repos=105615409,228681195,245246384,280546849,313382406&showPipelineDescriptions=false&showReleases=false) to roughly estimate when your Objective will have resources to be worked on (and here when I say "resource" I mean bandwidth from both Product Management and Engineering).
+
+The subsequent sections of this document describe what happens once it is your turn to be an Epic Lead. One note, at this point, I will start using the term Epic. An Epic is some demoable slice of work. They can be nested. Typically an Objective is its own Epic (thus the terms can be used interchangeably). However, an Objective can be broken down into smaller demoable chunks and each of those demoable chunks is also an Epic.
+
+## The Design Phase
+
+Once an Objective/Epic is ready to be worked on, the Epic enters the "Design Phase." During this phase the end goals are as follows:
+
+- The Objective is broken down into smaller chunks, each of which is a "Key Result" for that epic. These Key Results are written down in the OKR spreadsheet (reach out to your manager to figure out where this spreadsheet is if you don't know).
+- Each of the above smaller chunks has an Epic in Zenhub associated with it. The Product Manager will help you craft these issues.
```

Comment:
I create the issues with review by the Epic lead.

Reply:
Fixed. I also changed up this section a bit to delineate between "clarifying product requirements" which is driven by PM versus "refine the technical design" which is driven by EL.